client/app/root.tsx

```diff
@@ -26,14 +26,21 @@ import React from "react";
 
 export const loader = async ({ request }: LoaderFunctionArgs) => {
 	let user: User | undefined | null;
+	let notifications = 0;
 	try {
 		const userId = await checkAuth(request);
 		user = await prisma.user.findFirst({ where: { id: userId } });
+		notifications = user
+			? await prisma.notificationSuscribers.count({
+					where: { userId: user.id, read: false },
+			  })
+			: 0;
```

Comment:
This can be simplified as
```suggestion
		notifications = await prisma.notificationSuscribers.count({
					where: { userId: userId, read: false },
			  });
```